loader.lua

```diff
@@ -590,7 +592,9 @@ local function queryCard(cardID, forceNameQuery, forceSetNumLangQuery, onSuccess
 
     local language_code = getLanguageCode()
 
-    if forceNameQuery then
+    if getFirstPrint then
+        query_url = SCRYFALL_SEARCH_FIRSTPRINT_BASE_URL .. cardID.name:gsub(" ", "%%20") .. "%22"
```

Comment:
I think TTS handles escape codes for us, so we don't have to do the %3A, %20 stuff.

Also, let's just use `SCRYFALL_SEARCH_BASE_URL` here rather than defining a new one, since its the same scryfall endpoint.
```suggestion
        query_url = SCRYFALL_SEARCH_BASE_URL .. "not:reprint !\"" .. cardID.name .. "\""
```